src/Conference/ConferenceDetails.test.js

```diff
@@ -13,28 +19,45 @@ jest.mock('react-router-dom', () => {
 
 configure({ adapter: new Adapter() });
 
+beforeAll(() => {
+  jest.spyOn(console, 'error').mockImplementation((...args) => {
+    if (!args[0].includes('Warning: An update to %s inside a test was not wrapped in act')) {
+      consoleError(...args);
+    }
+  })
+})
+
 afterEach(() => {
   jest.clearAllMocks()
 });
 
 describe("ConferenceDetails", () => {
-  const mockData = [{
-    id: 1,
-    name: "Festival of Marketing",
-    dateTime: "2019-11-12T12:34:11Z",
-    city: "London",
-    description:
-      "From Festivalofmarketing.com: The Festival of Marketing is a unique experience where ambitious marketers can discover, learn, celebrate and shape the future together. As the largest global event dedicated to brand marketers, the Festival reflects the very nature of ...",
-    topic: "Marketing"
-  }];
-
-  it("renders conference details", () => {
+  const mockData = {
+    data: {
+      id: 1,
+      name: "Festival of Marketing",
+      dateTime: "2019-11-12T12:34:11Z",
+      city: "London",
+      description:
+        "From Festivalofmarketing.com: The Festival of Marketing is a unique experience where ambitious marketers can discover, learn, celebrate and shape the future together. As the largest global event dedicated to brand marketers, the Festival reflects the very nature of ...",
+      topic: "Marketing"
+    }
+  };
+
+  it("renders conference details", (done) => {
     expect.assertions(1);
 
-    useParams.mockReturnValue({ id: '1'})
- 
-    const component = shallow(<ConferenceDetails conferences={mockData} />);
-    
-    expect(component.debug()).toMatchSnapshot();
+    axios.get.mockResolvedValue(mockData);
+
+    useParams.mockReturnValue({ id: '1' });
+    let component;
+    component = mount(<ConferenceDetails />);
```

Comment:
Just a little post-refactor cleanup: you don't need to declare and assign in two lines anymore, you can now just assign to a const straight away :)